libraries/SITL/SIM_XPlane.cpp

```diff
@@ -532,14 +536,18 @@ bool XPlane::receive_data(void)
     accel_earth.z += GRAVITY_MSS;
     
     // the position may slowly deviate due to float accuracy and longitude scaling
-    if (loc.get_distance(location) > 4 || abs(loc.alt - location.alt)*0.01f > 2.0f) {
+    ftype delta_height;
```

Comment:
needs to be test flown over a long distance in XPlane

Reply:
Removed these XPlane patches.